atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -155,23 +156,10 @@ def get_output(self, worker_metadatas, buildroot_id):
         outputs = []
         output_file = None
 
-        if self.source_build:
-
-            registry = self.workflow.push_conf.docker_registries[0]
-
-            build_name = get_unique_images(self.workflow)[0]
-            pullspec = copy.deepcopy(build_name)
-            pullspec.registry = registry.uri
-
-            outputs, output_file = get_output(workflow=self.workflow, buildroot_id=buildroot_id,
-                                              pullspec=pullspec, platform=os.uname()[4],
-                                              source_build=True, logs=None)
-
-        else:
-            for platform in worker_metadatas:
-                for instance in worker_metadatas[platform]['output']:
-                    instance['buildroot_id'] = '{}-{}'.format(platform, instance['buildroot_id'])
-                    outputs.append(instance)
+        for platform in worker_metadatas:
+            for instance in worker_metadatas[platform]['output']:
+                instance['buildroot_id'] = '{}-{}'.format(platform, instance['buildroot_id'])
+                outputs.append(instance)
```

Comment:
Is this implementation specific to non-SourceContainer builds? Should it be in the appropriate subclass instead?

Reply:
Absolutely correct.